modules/fb_instant_articles_api_rules/fb_instant_articles_api_rules.rules_defaults.inc

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * Provide default rules integration to manage facebook instant article using API.
+ */
+
+/**
+ * Implements hook_default_rules_configuration().
+ */
+function fb_instant_articles_api_rules_default_rules_configuration() {
+  // Handling add/update node article as published in FB Instant Articles library.
+  $rule = rules_reaction_rule();
+  $rule->label = t('Add/update as published in FB Instant Articles library ' .
+                   'when a published Article node is saved');
+  $rule->active = FALSE;
+  $rule->event('node_presave')
+    ->condition('data_is', array('data:select' => 'node:status', 'value' => TRUE))
+    ->condition('data_is', array('data:select' => 'node:type', 'value' => 'article'))
```

Comment:
Hey yep, instead of checking "Article" content type (which is entirely my fault, since it was my suggestion to do this :wink: ) we could use this function since they already configured it elsewhere. Of course, with Rules people could still add any custom conditions they want, and with Drupal core actions they could add any custom triggers they want… but it would be nice to have this option in the default (example) rules.

So it would be nice to make a new Drupal core trigger (for the core Action), and also a Rules module condition.